CRM/Upgrade/Incremental/php/FourSeven.php

```diff
@@ -726,4 +727,19 @@ public function upgradeMailingFKs(CRM_Queue_TaskContext $ctx) {
     return TRUE;
   }
 
+  /**
+   * CRM-19100 - Alter Index and Type for Image URL
+   * @return bool
+   */
+  public static function alterIndexAndTypeForImageURL() {
+    $dao = CRM_Core_DAO::executeQuery("SHOW INDEX FROM civicrm_contact WHERE KEY_NAME = 'index_image_url'");
```

Comment:
suggest using this function instead https://github.com/civicrm/civicrm-core/blob/master/CRM/Core/BAO/SchemaHandler.php#L571